### PR TITLE
[KFSPTS-6941] Refactored code to consolidate methods generating dummy random data to be in module Utilities. Created new module KsbClientUtilities to house all KFS KSB client service setup and method calls that retrieve random data. Created new AFT parameters for items that were hard coded and currently could not be replaced by dummy or randomly looked-up data.

### PR DESCRIPTION
**Pull request by nkimble**
Code changes were verified locally and all AFT's ran to successful completion.

### CODE REVIEW

File: features/step_definitions/login_to_KFS.rb [hunk 1]
```diff
@@ -1,37 +1,39 @@
...
+  perform_backdoor_login(get_random_kfs_operations_user)
 end
 
 Given /^I am logged in as a KFS Fiscal Officer$/ do
   perform_backdoor_login(get_aft_parameter_value(ParameterConstants::DEFAULT_FISCAL_OFFICER))
 end
 
 Given /^I am logged in as a KFS User$/ do
-  perform_backdoor_login(get_random_principal_name_for_role('KFS-SYS', 'User'))
+  perform_backdoor_login(get_random_kfs_user)
```
Comment (db483) [inline features/step_definitions/login_to_KFS.rb:12]: I cannot find get_random_kfs_user defined in this commit. Is this an existing method?

**Review by db483 — COMMENTED** [review]
Changes look good to me but I had a question about one of the methods.

File: features/step_definitions/login_to_KFS.rb (hunk 1 above)
Comment (nkimble) [reply features/step_definitions/login_to_KFS.rb:12]: That is an existing method that was moved into new file    lib/kuality-kfs/ksb_client_utilities.rb     as part of this consolidation effort.
The method is located in that file starting on line 337.

Comment (db483) [reply features/step_definitions/login_to_KFS.rb:12]: Nancy helped me offline to find the ksb_client_utilities.rb and the new method. Thank you Nancy.

**Comment by db483** [comment]
This looks good to me. merging.

**Pull request merged**